portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,46 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
-  </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
-  </body>
-</html>
+
+<head>
+  <meta charset="UTF-8">
+  <title>Ibiyemi's Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+  <script src="script.js"></script>
+</head>
+
+<body>
+  <header>
+    <h1>
+      Ibiyemi Abiodun
+    </h1>
+  </header>
+  <section id="content">
+    <h2>
+      Hi, I'm Ibiyemi Abiodun!
+    </h2>
+    <p>
+      I'm a rising sophomore at Cornell, double majoring in electrical &
+      computer engineering, and computer science. I love cooking & baking,
+      karate, VFX, cinematography, and of course, coding.
+    </p>
+    <p>
+      I was born in
+      Nigeria, and I've lived all over the U.S., although when people ask I tell them I'm from Seattle because I lived in Bellevue & Kirkland for about 9 years of my 19-year life.
```

Comment:
It looks like GitHub defaults to showing 125 characters in a line in its diff view, so I'd suggest breaking this line off at 125 characters and putting the rest of the sentence on a new line. This makes it easier and nicer to read and is a good habit to get into (at Google, all lines are broken off at 100 characters).

There might be a way to set up a rule in your IDE so it shows a line at the 125 character mark while you're coding (like this: https://stackoverflow.com/questions/29968499/vertical-rulers-in-visual-studio-code).

Reply:
Ah, got it. I set my IDE to break at 80 lines, but I'll change it now.